src/sentry/ratelimits/utils.py

```diff
@@ -136,6 +139,7 @@ def get_organization_id_from_token(token_id: int) -> Any:
     # Return a random uppercase/lowercase letter to avoid collisions caused by tokens not being
     # associated with a SentryAppInstallation. This is a temporary fix while we solve the root cause
     if not installation:
+        logger.info("installation.not_found", extra={"token_id": token_id})
```

Comment:
I can use the token id here to found out what type of integration token this is (internal vs public) and see why we're not finding it when filtering from installs